test/remote/gateways/remote_mondido_test.rb

```diff
@@ -0,0 +1,1487 @@
+require 'test_helper'
+
+class RemoteMondidoTest < Test::Unit::TestCase
```

Comment:
@anderson-mondido can we get a shared AM test account to check in? It makes ongoing maintenance *way* easier for us maintainers.